docs/standards/sql.md

```diff
@@ -0,0 +1,24 @@
+# SQL Coding Standars
+
+When writing SQL developers should be concious to try to always ensure that all variables that are passed into the SQL are done in a safe and secure maner.
+
+In CiviCRM the best way is to use the inbuild paramatisation tools to do the job. 
+
+```php
+$name = 'John Smith';
+$genderId = 2;
+$result = CRM_Core_DAO::executeQuery("SELECT FROM civicrm_contact WHERE display_name like %1 AND gender = %2", array(
+  1 => array('%' . $name . '%', 'String'),
+  2 => array($genderId, 'Integer'),
```

Comment:
For non-SQL reasons, I'm not a fan of CiviCRM's storage of gender data. Maybe `is_opt_out` would be equally effective at demonstrating parameterization?